tests/backend_test.py

```diff
@@ -53,11 +53,10 @@ def test_iqm(authenticated_iqm_backend: IQMBackend) -> None:
     assert sum(counts.values()) == n_shots
 
 
-def test_invalid_cred(demo_settings_path: os.PathLike, demo_url: str) -> None:
+def test_invalid_cred(demo_url: str) -> None:
     with pytest.raises(Exception):
         _ = IQMBackend(
             url=demo_url,
-            settings=demo_settings_path,
             auth_server_url="https://cortex-demo.qc.iqm.fi/",
```

Comment:
I think the default `auth_server_url` is now "https://auth.demo.qc.iqm.fi", so I'm not sure if the exception is caused by invalid credentials or invalid auth server. We should check what error is raised here.

Reply:
To make sure this is checked properly I have now imported the exception from iqm to check it.